src/Bridges/Psr/Cache.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/**
+ * This file is part of the Tracy (https://tracy.nette.org)
+ * Copyright (c) 2004 David Grudl (https://davidgrudl.com)
+ */
+
+declare(strict_types=1);
+
+namespace Nette\Bridges\Psr;
+
+use DateInterval;
+use Generator;
+use Nette;
+use Psr;
+
+class Cache implements Psr\SimpleCache\CacheInterface
```

Comment:
`PsrCacheAdapter`?

Reply:
I saw the suffix in [tracy repository](https://github.com/nette/tracy/blob/master/src/Bridges/Psr/TracyToPsrLoggerAdapter.php), but this is really adapter. In this case, it is not adapter and Psr is contains in namespace.

What do you think `PsrCache` only?

Reply:
> In this case, it is not adapter

Why do you think it's not an adapter?

> Psr is contains in namespace.

See https://phpfashion.com/en/best-practices-for-namespaces-in-php#toc-1-a-class-should-have-a-descriptive-name-even-without-mentioning-the-ns

Reply:
Because I implement cache interface instead of storage interface. I have dependency on nette Storage and Cache is level up.  Feel free. I described only my way how i thinking about it.